pkg/controller/nodedaemon/reconcile.go

```diff
@@ -67,13 +67,5 @@ func (r *DaemonReconciler) Reconcile(request reconcile.Request) (reconcile.Resul
 		r.reqLogger.Info("daemonset changed", "daemonset.Name", ds.GetName(), "op.Result", opResult)
 	}
 
-	localProvisionerDSMutateFn := getLocalProvisionerDSMutateFn(request, tolerations, ownerRefs, nodeSelector, configMapDataHash)
```

Comment:
So why are we doing this only for `LocalVolumeSet` objects and not for `LocalVolume` objects?

Reply:
Lets fix this in a follow up.